projects/plugins/boost/app/assets/src/js/stores/critical-css-status-ds.ts

```diff
@@ -0,0 +1,95 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import z from 'zod';
+import { client, JSONSchema } from './data-sync-client';
+
+const CriticalCssErrorType = z.enum( [
+	'SuccessTargetError',
+	'UrlError',
+	'HttpError',
+	'UnknownError',
+	'CrossDomainError',
+	'LoadTimeoutError',
+	'RedirectError',
+	'UrlVerifyError',
+	'EmptyCSSError',
+	'XFrameDenyError',
+] );
+
+export type Critical_CSS_Error_Type = z.infer< typeof CriticalCssErrorType >;
```

Comment:
I prefer `CriticalCssErrorDetailsSchema` because it clearly separates Zod from a Type. But I hear what you're saying here - I'm not a fan of the naming here as well - consistency is key. This is just one of those things that snuck in in the storm of refactors :)

Reply:
I'm fine with `Schema`.